### PR TITLE
Clean & update: packages & Imports

### PR DESCRIPTION
**Pull request by Pratik-sys**
- added `.gitignore`
- updated all node packages
- set `index.js`  as default main file
- updated all images for slide-show
- fixed partials imports in `.ejs`

### CODE REVIEW

**Comment by Prathamesh404** [comment]
Are the changes checked thoroughly in local dev?

File: public/stylesheets/landingPage.css [hunk 1]
```diff
@@ -45,19 +45,19 @@ h1{
 }
```
Comment (Prathamesh404) [inline public/stylesheets/landingPage.css:46]: Is there a way to add images, instead of adding in CSS?

**Review by Prathamesh404 — APPROVED** [review]

**Comment by Pratik-sys** [comment]
> Are the changes checked thoroughly in local dev?

yes

File: public/stylesheets/landingPage.css (hunk 1 above)
Comment (Pratik-sys) [reply public/stylesheets/landingPage.css:46]: didn't get your point

**Pull request merged**